ui/css/design-system/_index.scss

```diff
@@ -1,6 +1,5 @@
 @forward 'attributes';
 @forward 'breakpoints';
 @forward 'colors';
-@forward 'pending-colors';
```

Comment:
What's the reason for removing this line? cc @garrettbear?

Reply:
The `pending-colors` file is actual CSS (as opposed to Sass vars, mixins, functions, etc) and should _never_ be `@use`d multiple times, which the `design-system` is. If it _is_ left here it will be copied into the built index.css file once for ever time `design-system` is `@use`d. So I had to move the file and then `@import`ed it once in ui/css/`index.scss` like the others.